app/src/main/java/com/github/sdp_begreen/begreen/fragments/ProfileDetailsFragment.kt

```diff
@@ -0,0 +1,102 @@
+package com.github.sdp_begreen.begreen.fragments
+
+import android.os.Build
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.ImageView
+import android.widget.ProgressBar
+import android.widget.RatingBar
+import android.widget.TextView
+import androidx.lifecycle.lifecycleScope
+import com.github.sdp_begreen.begreen.R
+import com.github.sdp_begreen.begreen.User
+import kotlinx.coroutines.launch
+
+
+// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+const val ARG_USER = "USER"
+
+/**
+ * A simple [Fragment] subclass.
+ * Use the [ProfileDetailsFragment.newInstance] factory method to
+ * create an instance of this fragment.
+ */
+class ProfileDetailsFragment : Fragment() {
+    var user: User? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        arguments?.let {
+            user = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+                it.getParcelable(ARG_USER, User::class.java)
+            }else {
+                it.getParcelable(ARG_USER)
+            }
+        }
+
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
+
+        val view = inflater.inflate(R.layout.fragment_profile_details, container, false)
+        val name: TextView = view?.findViewById(R.id.fragment_profile_details_profile_name) as TextView
```

Comment:
Here I think you don't need the `safe call` operator, as view will never be `null`

Reply:
Well seen!